com.unity.visualeffectgraph/Editor/Inspector/VFXSlotContainerEditor.cs

```diff
@@ -110,6 +110,8 @@ public virtual SerializedProperty DoInspectorGUI()
                     prop.Value.intValue = newValue;
                     modifiedSetting = prop.Value;
                 }
+
+                return modifiedSetting;
```

Comment:
Early exit here leads to the rest of the settings to be ignored.

Reply:
Really nice catch!
I didn't see that I added the early return in a for loop...